components/tooltip/tooltip.js

```diff
@@ -474,6 +476,7 @@ class Tooltip extends RtlMixin(LitElement) {
 		super.disconnectedCallback();
 		this._removeListeners();
 		window.removeEventListener('resize', this._onTargetResize);
+		document.body.removeEventListener('d2l-tooltip-show', this._onTooltipShowOther);
```

Comment:
I think this is already removed in `_removeListeners()`?

Reply:
Na... this one is different, for the new listener that I'm adding when the tooltip is shown. Normally it would be removed when the tooltip is hidden, but by off-chance the tooltip was disconnected while still being shown, this just makes sure we're not leaking memory. If the tooltip happens to be reconnected, it is initialized in the hidden state, and so upon showing again the handler would be once again wired up.

Reply:
Ah sorry my bad. The diff shown in GitHub made it look like you'd added a remove there as well but it was hiding a bunch of code so it wasn't _actually_ there.

<img width="481" alt="Screen Shot 2023-02-17 at 5 46 01 PM" src="https://user-images.githubusercontent.com/5491151/219811882-601ebfe2-74bb-44aa-a2ea-7852e4a8fdb9.png">